dataactcore/scripts/pipeline/load_sam_subaward.py

```diff
@@ -288,36 +295,29 @@ def parse_raw_subaward(raw_subaward_dict, data_type):
     return subaward_dict
 
 
-def store_subawards(sess, new_subawards, data_type):
-    """ Load the new subawards into the database
-
-        Args:
-            sess: database connection
-            new_subawards: dataframe of the new subaward data to be stored
-            data_type: either "contract" or "assistance"
-    """
+def store_subawards(sess: Session, new_subawards: pd.DataFrame, model: SAMSubgrant | SAMSubcontract) -> None:
+    """ Load the new subawards into the database"""
```

Comment:
StyleGuide: can you keep the `Args:` and just update `data_type` to model?

Additionally I know on USAS they utilize Python's type checking. We can have an tech debt effort to enforce throughout the code but for now can we remove it for consistency?